packages/core/lib/segments/attributes/subsegment.js

```diff
@@ -124,15 +117,15 @@ Subsegment.prototype.addPrecursorId = function(id) {
  */
 
 Subsegment.prototype.addAnnotation = function(key, value) {
-  if (!(isBoolean(value) || isString(value) || isFinite(value))) {
+  if (!(typeof value === 'boolean' || typeof value === 'string' || (typeof value === 'number' && isFinite(value)))) {
```

Comment:
Still has `isFinite` use

Reply:
`isFinite` should be available natively, see the following MDN reference for more info: [https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isFinite](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isFinite)

As far as I'm aware these should be completely interchangeable.

Reply:
I see, thanks for pointing that out!